FreeSurfer/custom/test_resample_fs

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+function invert_fsl
+{
+	echo convert_xfm -omat $2 -inverse $1
+	convert_xfm -omat $2 -inverse $1
+}
+function concat_fsl
+{
+	echo convert_xfm -omat $3 -concat $2 $1
+	convert_xfm -omat $3 -concat $2 $1
+}
+
+wd=/media/myelin/brainmappers/Connectome_Project/MishaLongitudinal/hcp
```

Comment:
Hardcoding a myelin path into a FreeSurfer/custom tool is not what we want.  If this doesn't make a necessary change to freesurfer's behavior, it shouldn't be in FreeSurfer/custom.  If it does, then we need to change it so it can get a reasonable value for `wd` when being run on other filesystems.

Reply:
This looks like another script that probably shouldn't be committed as I think it was for development and testing.

Reply:
removed